contributor-guide/modules/ROOT/pages/testing/continuous-integration.adoc

```diff
@@ -1,71 +1,94 @@
 = Continuous Integration
 
+
 The Boost project uses Continuous Integration (CI) to ensure the quality and integrity of its code. CI is the practice of merging all developers' working copies to a shared mainline several times a day. The main aim is to prevent integration problems, which are detected and can be fixed as early as possible.
 
 Boost uses several CI services for testing on different platforms and compilers. Many libraries use two or three of the systems described here, as does the https://github.com/boostorg/boost/tree/master[Super-project] itself.
 
 Note:: It is a requirement for a new library submission to Boost to include an appropriate CI system. Refer to the examples for each CI system to better understand what is involved.
 
+== New Library CI Framework
+
+A recommended process to start a new Boost library is to clone the contents of the https://github.com/boostorg/boost-ci/tree/master[boost-ci] repo. This repo contains the basic CI framework for a new library. Clone it, then adjust and edit it appropriately.
+
+This repository contains scripts that enable CI with:
+
+* <<GitHub Actions>>
+* <<Drone>>
+* <<Travis CI>>
+* <<AppVeyor>>
+* <<Codecov.io>>
+* <<Azure Pipelines>>
+* <<Coverity Scan>>
+
+For step-by-step processes, and tables of supported compilers, refer to the https://github.com/boostorg/boost-ci/blob/master/README.md[boost-ci README.md].
+
 == GitHub Actions
 
-Boost has been incorporating GitHub Actions into its testing workflows. This is a Continuous Integration/Continuous Deployment (CI/CD) system platform provided directly by GitHub. It can run tests on a variety of platforms and configurations. Here's a basic outline of how GitHub Actions works for Boost:
+Boost has been incorporating https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions[GitHub Actions] into its testing workflows. This is a Continuous Integration/Continuous Deployment (CI/CD) system platform provided directly by GitHub. It can run tests on a variety of platforms and configurations. Here's a basic outline of how GitHub Actions works for Boost:
```

Comment:
```suggestion
Boost has been incorporating https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions[GitHub Actions] into its testing workflows. This is a CI/CD system platform provided directly by GitHub. It can run tests on a variety of platforms and configurations. Here's a basic outline of how GitHub Actions works for Boost:
```

Reply:
done